post-processor/checksum/post-processor_test.go

```diff
@@ -0,0 +1 @@
+package checksum
```

Comment:
Can we include a test here? You can use the file builder to copy a file with a known checksum.